README.md

```diff
@@ -1,5 +1,62 @@
-# Northcoders News API
+# News API (News Space)
+
+> This project was built as part of the backend for the News Space app hosted on Heroku. The RESTful API provides the end-user with an interface to interact with the databases, and retrieve articles, comments, topics and users.
+> Live demo [_here_](https://news-space.herokuapp.com/).
+
+## Table of Contents
+
+- [General Info](#general-information)
+- [Technologies Used](#technologies-used)
+- [Setup](#setup)
+- [Contact](#contact)
+
+## General Information
+
+The RESTful API, through the various available endpoints, relays information to the end-user from the databases which store data entries about articles, comments, topics, and users. For example, the end-user may retrieve information about a selected article when an article id parameter has been inputted in the endpoint (‘https://news-space.herokuapp.com/api/articles/2’ will retrieve information about the article that has article id = 2). The API also accept queries in the request, such as ‘topic’ and ‘sort_by’. The former filters the retrieved array of articles by the topic referenced, and the latter sorts the retrieved array of articles by valid columns in the relational database for articles.
+
+## Technologies Used
+
+- node.js
+- Express (4.18.1)
+- PostgreSQL (14)
+
+## Setup
+
```

Comment:
All of these would be installed if someone just ran the command `npm install` or `npm i` - you can remove points 3-7. Instead add the steps where they'd need to add .env files and then run the tests